packages/terra-worklist-data-grid/src/ColumnHeaderCell.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { injectIntl } from 'react-intl';
+import ThemeContext from 'terra-theme-context';
+import IconUp from 'terra-icon/lib/icon/IconUp';
+import IconDown from 'terra-icon/lib/icon/IconDown';
+import IconError from 'terra-icon/lib/icon/IconError';
+import WorklistDataGridPropTypes from './proptypes/WorklistDataGridPropTypes';
+import './_elementPolyfill';
+import styles from './ColumnHeaderCell.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Data for columns. By default, columns will be presented in the order given.
+   */
+  column: WorklistDataGridPropTypes.columnShape.isRequired,
+  /**
+   * String that specifies the default width for columns in the grid. Any valid CSS width value is accepted.
+   */
+  width: PropTypes.string.isRequired,
+  /**
+   * String that specifies the column height. Any valid CSS height value accepted.
+  */
+  headerHeight: PropTypes.string.isRequired,
+  /**
+   * Function that is called when a selectable header cell is selected. Parameters: `onColumnSelect(columnId)`
+   */
+  onColumnSelect: PropTypes.func,
+  /**
+   * @private
+   * Object containing intl APIs
+   */
+  intl: PropTypes.shape({ formatMessage: PropTypes.func }),
+};
+
+const ColumnHeaderCell = (props) => {
+  const {
+    column,
+    width,
+    headerHeight,
+    onColumnSelect,
```

Comment:
What happens when onColumnSelect is undefined? Does it break or throw an error/warning?

Reply:
Nothing.  This is not required to be passed by consumers.  Consumers would only pass if they want to handle the action.